Pod/Classes/SideMenuTransition.swift

```diff
@@ -114,7 +114,7 @@ open class SideMenuTransition: UIPercentDrivenInteractiveTransition, UIViewContr
             view.transform = transform
             if velocity >= 100 || velocity >= -50 && abs(distance) >= 0.5 {
                 // bug workaround: animation briefly resets after call to finishInteractiveTransition() but before animateTransition completion is called.
-                if ProcessInfo().operatingSystemVersion.majorVersion == 8 && percentComplete > 1 - CGFloat(FLT_EPSILON) {
+                if ProcessInfo().operatingSystemVersion.majorVersion == 8 && percentComplete > 1 - CGFloat.ulpOfOne {
```

Comment:
I didn't check the minimum on that, my mistake, so no, my only goal was to get it compiling clean in Xcode 8.3. I could add an internal extension to `CGFloat` for versions of swift lower than 3.0 that returns the old `FLT_EPSILON` value?